src/format/strftime.rs

```diff
@@ -492,9 +503,12 @@ impl<'a> StrftimeItems<'a> {
 
 #[cfg(test)]
 mod tests {
+    use super::StrftimeItems;
```

Comment:
(As elsewhere, except I'd be okay with importing some often-used variants inside `tests` if they're used by multiple tests.)